modules/appeals_api/app/controllers/appeals_api/notice_of_disagreements/v0/notice_of_disagreements_controller.rb

```diff
@@ -30,13 +33,40 @@ class NoticeOfDisagreementsController < AppealsApi::V2::DecisionReviews::NoticeO
       ]
     }.freeze
 
+    def show
+      nod = AppealsApi::NoticeOfDisagreement.select(ALLOWED_COLUMNS).find(params[:id])
+      nod = with_status_simulation(nod) if status_requested_and_allowed?
+
+      render_notice_of_disagreement(nod)
+    rescue ActiveRecord::RecordNotFound
+      render_notice_of_disagreement_not_found(params[:id])
```

Comment:
The `show` action has not changed in behavior, but I updated it here because it relies on `render_notice_of_disagreement` and `render_notice_of_disagreement_not_found`, which were updated with these changes.